components/organisms/TopNav/top-nav.tsx

```diff
@@ -21,7 +25,7 @@ const TopNav: React.FC = () => {
             <Link className="text-sm text-light-slate-10" href={"/hub/insights"}>
             Insights Hub
             </Link>
-            <Link className="text-sm text-light-slate-10" href={"/javascript/dashboard/filter/recent"}>
+            <Link className="text-sm text-light-slate-10" href={"/"+userInterest+"/dashboard/filter/recent"}>
```

Comment:
Let's use a template literal here instead

```suggestion
            <Link className="text-sm text-light-slate-10" href={`${userInterest}/dashboard/filter/recent`}>
```